install_config/cfme/container_provider.adoc

```diff
@@ -0,0 +1,180 @@
+[[install-config-cfme-cloud-provider]]
+= Enabling Container Provider Integration
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+ifdef::openshift-enterprise[]
+:cloudforms: Red Hat CloudForms
+endif::[]
+ifdef::openshift-origin[]
+:cloudforms: ManageIQ
+endif::[]
+
+toc::[]
+
+== Overview
+
+After deploying {cloudforms} on {product-title} as described in
+xref:installing.adoc#install-config-cfme-installing[Running the Installer],
+there are two methods for enabling container provider integration. You can
+manually add {product-title} as a container provider, or you can try the
+playbooks included with this role.
+
+[[cfme-enabling-container-provider-manual]]
+== Enabling Manually
+
+See the following {cloudforms} documentation for steps on manually adding your
+{product-title} cluster as a container provider:
+
+ifdef::openshift-origin[]
+- link:http://manageiq.org/docs/reference/latest/doc-Managing_Providers/miq/#containers-providers[Container Providers]
+endif::[]
+ifdef::openshift-enterprise[]
+- link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html/integration_with_openshift_container_platform/[Integration with OpenShift Container Platform]
+endif::[]
+
+[[cfme-enabling-container-provider-automatically]]
+== Enabling Automatically
+
+Automated container provider integration can be accomplished using the
+playbooks included with this role.
+
+This playbook:
+
+. Gathers the necessary authentication secrets.
+. Finds the public routes to the {cloudforms} application and the cluster API.
+. Makes a REST call to add the {product-title} cluster as a container provider.
+
+To run the container provider playbook:
+
+----
+# ansible-playbook -v [-i /path/to/inventory] \
+ifdef::openshift-origin[]
+    playbooks/byo/openshift-management/add_container_provider.yml
+endif::[]
+ifdef::openshift-enterprise[]
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-management/add_container_provider.yml
+endif::[]
+----
+
+[[cfme-enabling-container-provider-multiple]]
+== Multiple Container Providers
+
+As well as providing playbooks to integrate your current {product-title} cluster
+into your {cloudforms} deployment, this role includes a Technology Preview
```

Comment:
link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] script, which

Reply:
Ooh yeah, I also meant to add the boilerplate Tech Preview blurb/box here.

Reply:
@adellape @ahardin-rh I just reviewed the code and there is reason we need to keep it labeled as *Tech Preview* anymore. That was probably a hold-over from when I was first writing up the playbook. I'm going to update that in the upstream README, too.

Reply:
:+1: